dms/models/dms_access_groups.py

```diff
@@ -39,74 +49,100 @@ class DmsAccessGroups(models.Model):
         store=True,
         recursive=True,
     )
-
     directory_ids = fields.Many2many(
-        comodel_name="dms.directory",
-        relation="dms_directory_groups_rel",
         string="Directories",
-        column1="gid",
-        column2="aid",
-        auto_join=True,
-        readonly=True,
+        comodel_name="dms.directory",
+        readonly=False,
```

Comment:
Why changing to `False`? And any case, it's the default value, so you can remove it.